### ISSUE TITLE
Add limited token support

### ISSUE THREAD

**Issue by jchaffer** [genericallyloud/ludomatic5000#15]
When including text in a template, support a succinct markup that expands something like `[foo]` to `<span class="icon icon-foo"></span>`.

**Comment by genericallyloud** [comment]
Yeah, almost exactly this has been on my list of things to do since the beginning. I was originally thinking <foo> but not sure if that would be confusing with HTML.

**Comment by jchaffer** [comment]
Maybe make it explicit so it doesn't become confusing? I could see adding a dictionary of replacements into ludo.json, so it's both configurable and 100% clear to users that the substitution will happen.

Regardless, this would be super-helpful, and would be the first feature to elevate this over my previous Photoshop workflow, since there's no way to do inline icons there.
